pytest.ini

```diff
@@ -0,0 +1,6 @@
+# pytest.ini
+[pytest]
+minversion = 6.0
+addopts = -ra -q
```

Comment:
to always show detailed info on skipped and xfailed tests, as well as have terser “dot” progress output

Reply:
This is useful, add it in config file itself plz

Reply:
We can enforce pep8 and flakes as well in options. 
`addopts = -ra -q --pep8 --flake` 
But that will required lot of cosmetic changes in files to pass the tests.

Reply:
Agree. But it will force the entire file to be up to the standard which is not easy given the size of our source files. We can enforce this option once all existing coding standard issues are addressed in our code base. @cdeshmukh pytest.ini itself is another config file.

Reply:
yes lets not enable pep8 check by default, we had tried this 2 years back, it needs dedicated & big efforts

>@cdeshmukh pytest.ini itself is another config file.

what does this mean. I meant add that comment in file explaining the config

Reply:
I added the description comment in config.